src/accel/cuda/GPUSolver.h

```diff
@@ -33,12 +36,6 @@
  *  group for a given Track */
 #define boundary_flux(t,pe2) (boundary_flux[2*(t)*(*polar_times_groups)+(pe2)])
 
-/** The value of 4pi: \f$ 4\pi \f$ */
-#define FOUR_PI 12.5663706143
-
-/** The values of 1 divided by 4pi: \f$ \frac{1}{4\pi} \f$ */
-#define ONE_OVER_FOUR_PI 0.0795774715
-
 /** The maximum number of polar angles to reserve constant memory on GPU */
 #define MAX_POLAR_ANGLES 10
```

Comment:
ditto. What do you think about adding `MAX_POLAR_ANGLES` and `MAX_AZIM_ANGLES` to `constants.h` (and perhaps adding `_GPU` at the end)?